### PR TITLE
feat(ui, repo): stream_chat_localizations [CDS-318]

### PR DESCRIPTION
**Pull request by xsahil03x**

### CODE REVIEW

File: packages/stream_chat_localizations/lib/src/stream_chat_localizations.dart [hunk 1]
```diff
@@ -6,7 +6,7 @@ import 'package:stream_chat_flutter/stream_chat_flutter.dart'
     show StreamChatLocalizations, User;
 
 part 'stream_chat_localizations_en.dart';
-
+part 'stream_chat_localizations_fr.dart';
```
Comment (xsahil03x) [inline packages/stream_chat_localizations/lib/src/stream_chat_localizations.dart:9]: @sachaarbonel I see what you did there :eyes:

File: packages/stream_chat_localizations/lib/src/stream_chat_localizations_fr.dart [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+  String _getDay(DateTime dateTime) {
+    final now = Jiffy(DateTime.now());
+    final date = Jiffy(dateTime);
+
+    if (date.isSame(now, Units.DAY)) {
+      return "aujourd'hui";
+    } else if (now.diff(date, Units.HOUR) < 24) {
+      return 'hier';
+    } else {
+      return 'le ${date.MMMd}'; //hmm lookup french format d/MM/yyyy
```
Comment (xsahil03x) [inline packages/stream_chat_localizations/lib/src/stream_chat_localizations_fr.dart:242]: @sachaarbonel Jiffy will do it automatically

Comment (sachaarbonel) [reply packages/stream_chat_localizations/lib/src/stream_chat_localizations_fr.dart:242]: ohh nice

File: packages/stream_chat_localizations/lib/src/stream_chat_localizations_fr.dart [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+  @override
+  String get allowGalleryAccessMessage => "Autoriser l'accès à votre galerie";
+
+  @override
+  String get flagMessageLabel => 'Signaler un message';
+
+  @override
+  String get flagMessageQuestion =>
+      'Voulez-vous envoyer une copie de ce message à une'
+      '\modérateur pour une enquête plus approfondie ?';
```
Comment (xsahil03x) [inline packages/stream_chat_localizations/lib/src/stream_chat_localizations_fr.dart:174]: @sachaarbonel is this a mistake?

```suggestion
      '\nmodérateur pour une enquête plus approfondie ?';
```

Comment (sachaarbonel) [reply packages/stream_chat_localizations/lib/src/stream_chat_localizations_fr.dart:174]: well catch

**Comment by imtoori** [comment]
@xsahil03x let's add something in the readme regarding support locales and that it's possible to use custom ones

**Review by imtoori — APPROVED** [review]
LGTM

**Pull request merged**